Source/SwiftLintFramework/Rules/Idiomatic/RandomRule.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+import SourceKittenFramework
+
+public struct RandomRule: OptInRule, ConfigurationProviderRule, AutomaticTestableRule {
+
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static var description = RuleDescription(
+        identifier: "random",
```

Comment:
maybe `legacy_random` would be a better name? We already have some `legacy_*` rules

Reply:
Sounds good, thanks! I’ll correct everything this weekend